lib/services/remote_sync_service.dart

```diff
@@ -212,6 +212,11 @@ class RemoteSyncService {
             " files were ignored for upload");
       }
     }
+    if (filesToBeUploaded.isEmpty) {
```

Comment:
Should do this even if it's not empty?

Reply:
I was thinking of that, was just worried about performance hit. This felt okay to me, given the possible low usage of this flow.